projects/js-packages/publicize-components/src/components/share-buttons/styles.module.scss

```diff
@@ -32,3 +32,14 @@ $whatsapp: #2ab318;
 		color: var(--jp-black);
 	}
 }
+
+.container {
+	display: flex;
+}
+
+.label.label {
+	font-size: 15px;
+	font-weight: 400;
+	line-height: 40px;
```

Comment:
Similarly, the icon size in the design is about 24px, and the line height of ends up being 36px, with margins/padding after setting this to 24px also.

Although fiddling about with that, I think it does make the icons quite small, but does feel better than the current size

<img width="379" alt="image" src="https://github.com/Automattic/jetpack/assets/96462/bbfa41b2-e412-43dc-9a84-5b7929e42be8">

Reply:
Addressed in 94fc991b86